README.md

```diff
@@ -222,16 +217,54 @@ $ createdb `whoami`
 $ make test_serial
 ```
 
-##### Errors related to too many _open files_
+##### Errors related to Capybara in feature tests
+You may need to install _chromedriver_ or your chromedriver may be the wrong version (`$ which chromedriver && chromedriver --version`).
+
+chromedriver can be installed using [Homebrew](https://formulae.brew.sh/cask/chromedriver) or [direct download](https://chromedriver.chromium.org/downloads). The version of chromedriver should correspond to the version of Chrome you have installed `(Chrome > About Google Chrome)`; if installing via Homebrew, make sure the versions match up. After your system recieves an automatic Chrome browser update you may have to upgrade (or reinstall) chromedriver.
+
+If `chromedriver -v` does not work you may have to [allow it](https://stackoverflow.com/questions/60362018/macos-catalinav-10-15-3-error-chromedriver-cannot-be-opened-because-the-de) with `xattr`.
+
+##### Errors related to _too many open files_
 You may receive connection errors similar to the following:
 
 `Failed to open TCP connection to 127.0.0.1:9515 (Too many open files - socket(2) for "127.0.0.1" port 9515)`
 
-Running the following, _prior_ to running tests, may solve the problem:
+You are encountering you OS's [limits on allowed file descriptors](https://wilsonmar.github.io/maximum-limits/). Check the limits with both:
+* `ulimit -n`
+* `launchctl limit maxfiles`
+
+Try this to increase the user limit:
 ```
 $ ulimit -Sn 65536 && make test
 ```
 To set this _permanently_, add the following to your `~/.zshrc` or `~/.bash_profile` file, depending on your shell:
 ```
 ulimit -Sn 65536
 ```
+
+If you are running MacOS, you may find it is not taking your revised ulimit seriously. [Insist by](https://medium.com/mindful-technology/too-many-open-files-limit-ulimit-on-mac-os-x-add0f1bfddde) adding the following file at `/Library/LaunchDaemons/limit.maxfiles.plist` and set it to be owned by root:
```

Comment:
😭 I wish we didn't have to resort to this....

but as long as we're here, can we make it easier to copy paste? Like a `chown` command to copy? and maybe bonus like one of these `cat` commands to write to the file

```bash
cat > /Library/LaunchDaemons/limit.maxfiles.plist <XML
<?xml ....
XML
```

Reply:
A nice idea! Sadly I think that using `cat >` in a directory owned by root would require interactive `sudo -i` login, which I have a bad feeling about. The easiest way to do it in my opinion is to use `sudo` to launch the editor, then paste the file contents in. I've changed the README to reflect this!

Reply:
Since this requires root access are people with GFEs allowed to do this?